mod/profile.php

```diff
@@ -233,7 +234,7 @@ function profile_content(App $a, $update = 0) {
 			FROM `item` INNER JOIN `contact` ON `contact`.`id` = `item`.`contact-id`
 			AND `contact`.`blocked` = 0 AND `contact`.`pending` = 0
 			WHERE `item`.`uid` = %d AND `item`.`visible` = 1 AND
-			(`item`.`deleted` = 0 OR item.verb = '" . ACTIVITY_LIKE ."'
+			(`item`.`deleted` = 0 OR item.verb = '" . ACTIVITY_LIKE . "'
```

Comment:
It would be better if these constants had been treated like all regular SQL parameters.

Reply:
I disagree, constants are the only SQL parameters we have complete control over, so we don't have to guess what parameter goes where in the query without any security impact.

Reply:
Some attacker with partly write access could replace the constants. Or possibly there are other attack vectors where constants could be replaced by variables.

Reply:
If an attacker gets write access to your server files, it's game over anyway, no amount of SQL escaping will prevent that.

Reply:
I know. But I just want to be cautious. During the reworking of all database functions we really should take care about this.